### ISSUE TITLE
"Serious,_Critical" name to be changed and wrong country numbers

### ISSUE THREAD

**Issue by KixMan28** [yudeep-rajbhandari/covid-19-api#3]
Hi @yudeep-rajbhandari ! Than you for this API. I have pulled some data from your API into Home Assistant. The REST API method and the JSON it gives error on this data name due to the fact that it contains a "," (coma). Could you please change the name of this data to "Serious_Critical"? 

Could you please also check the country numbers, because according to my results the numbers don't match anymore. For example Hungary is 37 and not 69, as mentioned in the readme.md

Thank you for your understanding and support.

**Comment by CycleMichael** [comment]
The "Country Numbers" in the API appear to be not unique to the Country but are the ordinal numbers in the current "league table" of "Total Cases" at https://www.worldometers.info/coronavirus/ so they will change over time as the relative position changes with the data. So for instance in the https://github.com/yudeep-rajbhandari/covid-19-api Readme.md "Kenya"  is 133, today in the data Kenya is obtained from https://covid-19-apis.herokuapp.com/update/country/83/

**Comment by yudeep-rajbhandari** [comment]
I have made necessary changes  and changed documentation to new country numbers, please have a look at it and close the issue if your complaint is fixed.

**Comment by CycleMichael** [comment]
@yudeep-rajbhandari 
_> I have made necessary changes and changed documentation to new country numbers, please have a look at it and close the issue if your complaint is fixed._

Thanks for quick response and update to country numbers.  A couple of queries:
I take this order can change as the order of total case numbers per country change.  Would it be possible to sort by Country alphabetically first or extract to a country code so that country assigned code/numbers remain static?
Second,  I just see discrepency with some numbers still, eg:
in the https://github.com/yudeep-rajbhandari/covid-19-api Readme.md "Comoros" is 183
but today data for Comoros is obttained by  https://covid-19-apis.herokuapp.com/update/country/182/

**Comment by KixMan28** [comment]
@yudeep-rajbhandari Thank you for the update. The "Serious_Critical" works now as intended. Regarding the country number would be good if you could set up a fixed country name/country code list and you would somehow automatize the country number changes in the background.

**Comment by KixMan28** [comment]
The contry numbers changed again. I needed to update my country's number (Hungary) to 38.

**Comment by hvalev** [comment]
I agree, it's really nice what you've done, but it's practically unusable in this form. I have to implement logic to parse country codes in the vicinity of where I'm expecting my country to be. By that point I might as well scrape the source website myself. I think the idea of @CycleMichael of using alphabetical ordering is great. If it's not possible or too complicated, you can hardcode a dictionary which maps country identifiers in the source website to country codes. That won't affect the order even if a new country springs into existence.

**Comment by scstraus** [comment]
Hvalvev, I certainly wouldn't say it's practically unusbale. The solution you posted on the thread linked in above comment solved all the problems (at least when the API is working).

**Comment by yudeep-rajbhandari** [comment]
Sorry for late fix , but I have made changes for numbering, please check and close the issue if itsdone.

**Comment by hvalev** [comment]
> 
> 
> Sorry for late fix , but I have made changes for numbering, please check and close the issue if itsdone.

Thank you @yudeep-rajbhandari for addressing this issue 👍 I ended up using an alternative way to get the numbers using the allUpdate endpoint from your API, but looking at the code changes, this should work! 

> 
> 
> Hvalvev, I certainly wouldn't say it's practically unusbale. The solution you posted on the thread linked in above comment solved all the problems (at least when the API is working).

Ah, the context of this issue is that countries changed their numerical indexing in the API that is supposed to identify them using that exact numerical indexing (https://covid-19-apis.herokuapp.com/update/country/{country}). The fix that I proposed in the other thread is using a different endpoint (https://covid-19-apis.herokuapp.com/allUpdate).
